libbeat/reader/multiline/multiline_config.go

```diff
@@ -29,15 +29,18 @@ type multilineType uint8
 const (
 	patternMode multilineType = iota
 	countMode
+	whilePatternMode
 
-	patternStr = "pattern"
-	countStr   = "count"
+	patternStr      = "pattern"
+	countStr        = "count"
+	whilePatternStr = "while-pattern"
```

Comment:
Could you please rename it to `while_pattern`?

Reply:
yes, no problem

Reply:
@kvch could you review it?